modules/message-lane/src/lib.rs

```diff
@@ -137,13 +142,23 @@ decl_error! {
 
 decl_storage! {
 	trait Store for Module<T: Trait<I>, I: Instance = DefaultInstance> as MessageLane {
+		/// Optional pallet owner. Pallet owner has a right to halt all pallet operations
+		/// and then resume it. If it is `None`, then there are no direct ways to halt/resume
+		/// pallet operations, but other runtime methods may still be used to do that (i.e.
+		/// democracy::referendum to update halt flag directly or call the `halt_operations`).
+		pub ModuleOwner get(fn module_owner) config(): Option<T::AccountId>;
```

Comment:
Shall we add ownership transfer function? Now we have to rely on `sudo/democracy` to alter that entry.
Something like
```rust
set_owner(origin, new_owner: Option<T::AcountId>) {
  ensure_owner_or_root::<T, I>(origin)?;
  ModuleOwner::put(new_owner);
}

Reply:
:) I have removed this function right before opening PR just because I was unable to guess when we'll need it. Most probably this account will be shared in some way and then (when the time will come) we will simply rely on sudo/democracy to transfer ownership to root? Or do you think it is better to have explicit function for this?

Reply:
> Most probably this account will be shared in some way

Yes, I assumed that as well.

> we will simply rely on sudo/democracy to transfer ownership to root

Here I actually assumed that WE would just send a transaction setting the owner to `None` - which would make `Root` the only owner. I prefer that cause it doesn't involve democracy - it's purely our decision when we want to get rid of the admin rights.

I'm fine with either though, I think `set_owner` is a bit more flexible though long-term: you can go with single key -> multi-sig -> democracy for instance.